pkg/analysis/completion_test.go

```diff
@@ -39,12 +39,12 @@ func TestSimpleCompletion(t *testing.T) {
 
 	f.Symbols("foo", "bar", "baz")
 
-	f.Document("")
-	result := f.a.Completion(f.doc, protocol.Position{})
+	doc := f.MainDoc("")
```

Comment:
a lot of the PR is changes like this to support `load`s in analysis tests

Reply:
Ugh yeah, lots of test rework. Thanks for cleaning up.